### PR TITLE
feat(rest) : Export Project Create Clearing Request

### PR DESCRIPTION
**Pull request by keerthi-bl**
### Description

Rest end point forExport Project Create Clearing Request

#### How to reproduce

Closes : #2461 

This PR should be merged after #2442

### CODE REVIEW

**Comment by nikkuma7** [comment]
Test successful.

**Comment by GMishx** [comment]
@keerthi-bl please rebase with latest main

File: rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/project/ProjectController.java [hunk 1]
```diff
@@ -141,8 +135,8 @@
 import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
 import org.eclipse.sw360.datahandler.thrift.users.RequestedAction;
 
-import jakarta.servlet.http.HttpServletRequest;
-import jakarta.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
```
Comment (GMishx) [inline rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/project/ProjectController.java:139]: Please use `jakarta` not `javax`

File: rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/report/SW360ReportService.java [hunk 2]
```diff
@@ -5,28 +5,77 @@
...
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.function.Function;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
+import javax.servlet.http.HttpServletRequest;
```
Comment (GMishx) [inline rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/report/SW360ReportService.java:22]:
```suggestion
import jakarta.servlet.http.HttpServletRequest;
```

**Review by GMishx — CHANGES_REQUESTED** [review]
Please rebase with latest main

**Comment by GMishx** [comment]
@keerthi-bl #2442 was merged. Please rebase this branch for review and test.

**Review by GMishx — APPROVED** [review]
Changes looks good.

**Pull request merged**